locales/en-US/app.ftl

```diff
@@ -728,21 +728,47 @@ TrackMemoryGraph--relative-memory-at-this-time = relative memory at this time
 TrackMemoryGraph--memory-range-in-graph = memory range in graph
 TrackMemoryGraph--operations-since-the-previous-sample = operations since the previous sample
 
-## TrackPowerGraph
+## TrackPower
 ## This is used to show the power used by the CPU and other chips in a computer,
 ## graphed over time.
 ## It's not displayed by default in the UI, but an example can be found at
 ## https://share.firefox.dev/3a1fiT7.
 
-# This is used in the tooltip when the power value uses the Watt unit.
+# This is used in the tooltip when the power value uses the watt unit.
 # Variables:
 #   $value (String) - the power value at this location
-TrackPowerGraph--tooltip-power-watt = Power: <em>{ $value } W</em>
+TrackPower--tooltip-power-watt = { $value } W
+  .label = Power
 
-# This is used in the tooltip when the power value uses the Milliwatt unit.
+# This is used in the tooltip when the instant power value uses the milliwatt unit.
 # Variables:
 #   $value (String) - the power value at this location
-TrackPowerGraph--tooltip-power-milliwatt = Power: <em>{ $value } mW</em>
+TrackPower--tooltip-power-milliwatt = { $value } mW
+  .label = Power
+
+# This is used in the tooltip when the sum of power value for the current range
+# uses the watt-hour unit.  Variables:
+#   $value (String) - the power value at this location
+TrackPower--tooltip-power-used-in-range-watthour = { $value } Wh
+  .label = Power used in range
+
+# This is used in the tooltip when the sum of power value for the current range
+# uses the milliwatt-hour unit.  Variables:
+#   $value (String) - the power value at this location
+TrackPower--tooltip-power-used-in-range-milliwatthour = { $value } mWh
+  .label = Power used in range
+
+# This is used in the tooltip when the sum of power value for the current
+# preview selection uses the watt-hour unit.  Variables:
+#   $value (String) - the power value at this location
+TrackPower--tooltip-power-used-in-preview-watthour = { $value } Wh
+  .label = Power used in temporary selection
```

Comment:
What is the difference between "range" and "temporary selection"? Maybe change "temporary" to "current".

Reply:
Here is what I explain above:

> In the UI, there's a distinction between the "committed" range, and the preview selection. From the preview selection we can either click "+" to make it the "committed" range (and pass it full screen), or click elsewhere to unselect it. That's what I call the "temporary" selection. But I'm not especially happy about that either.

I used `current` instead of `temporary` like you suggested :-)